src/components/tools/networked-drawing.js

```diff
@@ -471,6 +475,9 @@ AFRAME.registerComponent("networked-drawing", {
     this.currentPointCount = 0;
     this.lineStarted = false;
     this.drawStarted = false;
+
+    AFRAME.scenes[0].systems["hubs-systems"].cursorTargettingSystem.setDirty();
```

Comment:
Do you think we should add a second API here that only updates the cursorTargettingSystem's knowledge of this specific drawing?

Reply:
Maybe, but turns out I don't actually need this line here at all.